src/UserException.java

```diff
@@ -0,0 +1,21 @@
+public class UserException extends Exception {
```

Comment:
Review existing exception and their names.
We can ussually understand why this exception was thrown based on its name

Your exception don't give any info about an exception